modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/LegacyRomanianStemmer.java

```diff
@@ -0,0 +1,744 @@
+/*
+ * @notice
+ * Generated by Snowball 2.0.0 - https://snowballstem.org/
+ *
+ * Modifications copyright (C) 2024 Elasticsearch B.V.
+ */
+
+package org.elasticsearch.analysis.common;
+
+import org.tartarus.snowball.Among;
+
+/**
+* This class implements the stemming algorithm defined by a snowball script.
+* NOTE: This is the RomanianStemmer used in Lucene 9
+*
+* @deprecated this class exists for backwards compatibility of indices created with Lucene 9
+*/
+@Deprecated
+@SuppressWarnings("checkstyle:DescendantToken")
+public class LegacyRomanianStemmer extends org.tartarus.snowball.SnowballStemmer {
+
```

Comment:
We should remove `Deprecated` and make this class package private.

Reply:
Sure np